packages/services/src/utils/helpers.ts

```diff
@@ -301,6 +301,11 @@ export const constructObject = (
   return data;
 };
 
+export const extractDomain = (url: string): string => {
```

Comment:
Can you provide the unit test of this function?

Reply:
Good catch!